### PR TITLE
Added 1 polearm, 1 two hander and 10 one handers

### PR DESCRIPTION
**Pull request by Erdertus**
Added weapons

Polearm: Great Long Axe
Two Hander: Large Falchion
One Hander: Arabian Cavalry Sword, Arabian Straight Sword, Kilij, Shamshir, Karabela, Noble Steel Scimitar, Solingen Sword, Heavy Arming Sword, Long Arming Sword & Decorated Thegn Blade
_____________________________________________________________________________________

Nami, everything but GLA has been tested in SP environment. GLA is made of same pieces as two-handed "Great Axe" or "Long Great Axe". I didn't touch offsets etc so it should just work but let me know if you want me to remove it until it can be tested <3

newweapons
3 new ones
arabcav

### CODE REVIEW

**Review by namidaka — APPROVED** [review]

File: items.json [hunk 1]
```diff
@@ -90775,6 +91663,146 @@
...
   },
+  {
+    "id": "crpg_kilij_h0",
+    "baseId": "crpg_kilij",
+    "name": "Kilij",
+    "culture": "Aserai",
+    "type": "OneHandedWeapon",
+    "price": 12893,
+    "weight": 1.77,
+    "tier": 13.9751415,
```
Comment (Yeldur) [inline items.json:91674]: Tier 13.9 - Please fix and bring down to tier match (presumably around 7.0 - 7.5 tier judging by higher tiers)

**Review by Yeldur — CHANGES_REQUESTED** [review]
Please address the requested, after this can be approved/merged

File: items.json (hunk 1 above)
Comment (Erdertus) [reply items.json:91674]: Tier has been adjusted (H0 blade weight wasn't in line)

**Review by Yeldur — APPROVED** [review]
Change requests have been addressed -> Approved

File: items.json (hunk 1 above)
Comment (Erdertus) [reply items.json:91674]: Couldn't hit T7.5 without messing with pommel/guard/handle also, T7.6 still inline though imo

**Review by Meowkov — APPROVED** [review]
Don't see any format or syntax errors, item usage is default without exclusions, no issues there.
Good to go.

**Comment by namidaka** [comment]
@Yeldur

**Comment by Erdertus** [comment]
@namidaka @Yeldur Rebasing is complete, I *think* I did it correctly

**Comment by Erdertus** [comment]
PR is fucked because it includes the changes from the sheathes/holster PR that broke a billion items - I assume I rebased wrong. Closing to recreate (kill me now).

**Pull request closed without merging**